README.md

```diff
@@ -26,6 +26,12 @@ the [LuaRocks](http://luarocks.org) package manager:
 $ luarocks install --local --only-deps pallene-dev-1.rockspec
 ```
 
+> As of now, LuaRocks is configured to use Lua 5.1 by default. However, Pallene
+> requires Lua 5.3 to run. Therefore, please configure LuaRocks to use Lua 5.3.
+> You can use the following command to configure LuaRocks to use Lua 5.3 when
+> compiling it:
+> `./configure --lua-version=5.3`
+
```

Comment:
Instead of this quote-like paragraph, you could write a section called `### Prerequisites`. In this section, you should mention that the user needs Lua 5.3 installed and then he/she should compile LuaRocks. It would be nice to provide links to where you can get Lua and LuaRocks. Only after that you should explain the issue with Lua 5.1. It is also important to notice that this issue should only happen when you have Lua 5.1 installed, which probably is the case for Ubuntu users.

Reply:
I've made the requested changes.